lib/yaml_elixir/exceptions.ex

```diff
@@ -3,5 +3,27 @@ defmodule YamlElixir.FileNotFoundError do
 end
 
 defmodule YamlElixir.ParsingError do
-  defexception message: "parsing error"
+  defexception [:line, :column, :type, message: "parsing error"]
+
+  @impl true
+  def message(%__MODULE__{message: message, line: nil, column: nil}) do
+    message
+  end
+  def message(%__MODULE__{message: message, line: line, column: column}) do
+    message <> " (line: #{line}, column: #{column})"
+  end
+
+  def from_yamerl({:yamerl_parsing_error, :error, human_readable_error, line, column, error_type, _token_being_parsed, _}) do
+    %__MODULE__{
+      message: to_string(human_readable_error),
+      line: line,
+      column: column,
+      type: error_type
+    }
+  end
```

Comment:
Please use mix formatter and Elixir styleguide. Functions have to be separated by an empty line.